sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2658,17 +2658,17 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       sql("select LTRIM(BOTH 'S' FROM 'SS abc S')").head
     }
     assert(ae1.getMessage contains
-      "The specified function ltrim doesn't support with option BOTH")
+      "The specified function LTRIM doesn't support with option BOTH")
     val ae2 = intercept[ParseException]{
       sql("select RTRIM(TRAILING 'S' FROM 'SS abc S')").head
     }
     assert(ae2.getMessage contains
-      "The specified function rtrim doesn't support with option TRAILING")
+      "The specified function RTRIM doesn't support with option TRAILING")
     val ae3 = intercept[ParseException]{
-      sql("select TRIM(WINDOW 'S' FROM 'SS abc S')").head
+      sql("select TRIM(OVER 'S' AND 'SS abc S')").head
```

Comment:
I was thinking to try to use different keywords to see whether make the exception different. I forgot to change it back to WINDOW or remove this test case.